appveyor.yml

```diff
@@ -22,15 +38,26 @@ before_build:
   - nuget sources add -name funcparser -source https://ci.appveyor.com/nuget/ospsuite-funcparser 
   - nuget sources add -name cvodes -source https://ci.appveyor.com/nuget/ospsuite-simmodel-solver-cvodes
   - ps: (get-content src\OSPSuite.SimModelNative\version.h) | foreach-object {$_ -replace "RELEASE 0", "RELEASE $env:RELEASE" -replace "MAJOR 0", "MAJOR $env:MAJOR" -replace "MINOR 0", "MINOR $env:MINOR" -replace "BUILD 0", "BUILD $env:APPVEYOR_BUILD_NUMBER"} | set-content src\OSPSuite.SimModelNative\version.h
+  - sh: nuget install packages.config -OutputDirectory packages -ExcludeVersion
   - dotnet restore
   - dotnet add tests/OSPSuite.SimModel.Tests/OSPSuite.SimModel.Tests.csproj package Appveyor.TestLogger --version 2.0.0
+  # linux: remove all native c++ projects and test application from the solution
+  - sh: dotnet sln OSPSuite.SimModel.sln remove src/OSPSuite.SimModelNative/OSPSuite.SimModelNative.vcxproj
+  - sh: dotnet sln OSPSuite.SimModel.sln remove src/OSPSuite.SysTool/OSPSuite.SysTool.vcxproj
+  - sh: dotnet sln OSPSuite.SimModel.sln remove src/OSPSuite.XMLWrapper/OSPSuite.XMLWrapper.vcxproj
+  - sh: dotnet sln OSPSuite.SimModel.sln remove tests/TestAppNetCore/TestAppNetCore.csproj
+
+install:
+  #linux: install libxml2
+  - sh: sudo apt-get install libxml2-dev --assume-yes 
  
 after_build:
   - cmd: nuget pack src\OSPSuite.SimModel\OSPSuite.SimModel.nuspec -version %app_version%
+  - sh: nuget pack src/OSPSuite.SimModel/OSPSuite.SimModel.Ubuntu18.nuspec -version $app_version
   - ps: Get-ChildItem .\OSPSuite.SimModel.*.nupkg | % { Push-AppveyorArtifact $_.FullName -FileName $_.Name }
 
 test_script:
-  - ps: dotnet test --no-build --no-restore --logger:Appveyor
+  - ps: dotnet test --no-build --no-restore --logger:Appveyor --configuration:Release 
```

Comment:
seems that per default the Debug-configuration is used by dotnet test.
Will adjust it for FuncParser and CVODE-Solver as well.